CONTRIBUTING.md

```diff
@@ -27,53 +32,49 @@ This is a good way to introduce yourself
 and to meet some of our community members.
 
 1.  If you do not have a [GitHub][github] account,
-    you can [send us comments by email][contact].
-    However,
-    we will be able to respond more quickly if you use one of the other methods described below.
+you can [send us comments by email][contact].
+However, we will be able to respond more quickly if you use one of the other 
+methods described below.
```

Comment:
Starting here, large part of the diff seem to be "only" about line lengths, wrapping & indentation. I'm having trouble picking out the content changes and would prefer to keep at least the indentation. The [style guide doesn't mention it](https://www.google.com/search?q=indent+site:http://carpentries.github.io/lesson-example), but I'll add a commit to see whether the review is easier then.